website/Application.js

```diff
@@ -39,6 +51,10 @@ class Application extends Component {
         </ul>
 
         <p>
+          <label className={styles.Label}>
+            <input type='checkbox' onChange={(e) => (timeoutUpdater(e.target.checked, udpateArray, udpateMap))}/>
+            Auto update (every 10 ms)
+          </label>
```

Comment:
I'm not opposed to some kind of automation but I think it should be far less frequent than 10ms.

nit: It would also be nice if it used all 4 methods rather than just 2.

Reply:
fixed) 1s